DESCRIPTION

```diff
@@ -38,11 +38,15 @@ Imports:
     renv,
     rlang,
     rmarkdown,
+    tidyr,
```

Comment:
are you certain `tidyr` is not used anywhere else in this repo? this seems a bit dangerous

Reply:
Out of curiosity, why would it matter if `tidyr` is used anywhere else in the repo?

Reply:
weird, I thought this was a removal when I read it